Tools/clinic/clinic.py

```diff
@@ -5335,8 +5374,15 @@ def at_critical_section(self, *args: str) -> None:
         self.critical_section = True
 
     def at_getter(self) -> None:
+        if self.kind in (GETTER, SETTER):
+            fail("@getter and @setter can not be declared at once")
```

Comment:
How about one of these? (Ditto for the `fail(...)` below.)

```suggestion
            fail("only one of @getter or @setter can be used")
```

(I'm not super enthusiastic about my suggestion; perhaps @AlexWaygood can cook up something nicer 😃)